<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rejestracja</class>
 <widget class="QDialog" name="Rejestracja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rejestracja</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Wpisz nick:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="poleNick"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Wpisz hasło:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="polePass"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ponownie wpisz hasło:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="polePass2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushRejestruj">
      <property name="text">
       <string>Rejestruj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushAnuluj">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
